Gemfile

```diff
@@ -3,7 +3,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 gem 'faker', '~> 3.2.1'
 gem 'jwt_auth', git: 'https://github.com/performant-software/jwt-auth.git', tag: 'v0.1.2'
-gem 'resource_api', git: 'https://github.com/performant-software/resource-api.git', tag: 'v0.5.5'
+#gem 'resource_api', git: 'https://github.com/performant-software/resource-api.git', tag: 'v0.5.5'
+gem 'resource_api', path: '../resource-api'
```

Comment:
I know this is just part of the process but I'd feel weird approving this PR without making a note of this line.